File: 1-two-sum/1-two-sum.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        dictionary = {index:target - num2 for index, num2 in zip(range(len(nums)), nums)}

        for index, required in dictionary.items():
            try:
                output = [index, nums.index(required)]
                if output[0] == output[1]:
                    assert balls
            except:
                pass
            else:
                return output

File: 1-two-sum/1-two-sum.java
class Solution {
    public int[] twoSum(int[] nums, int target) {
        HashMap<Integer, Integer> seen = new HashMap<>();
        int[] values = {-1, -1};
        
        for (int i = 0; i < nums.length; i++) {
            try {
                values[0] = seen.get(target - nums[i]);
                values[1] = i;
                break;
            } catch (Exception e) {
                seen.put(nums[i], i);
            }
        }
        
        return values;
    }
}